Octokit.Tests.Integration/Clients/RepositoryCommentsClientTests.cs

```diff
@@ -377,6 +378,29 @@ public TheCreateMethod()
             Assert.NotNull(retrieved);
         }
 
+        [IntegrationTest]
+        public async Task CanGetReactionPayload()
+        {
+            var commit = await SetupCommitForRepository(_github);
+
+            var comment = new NewCommitComment("test");
+
+            var result = await _github.Repository.Comment.Create(_context.Repository.Id,
+                commit.Sha, comment);
+
+            var reaction = await _reactionsClient.CommitComment.Create(_context.RepositoryOwner, _context.RepositoryName, result.Id, new NewReaction(ReactionType.Confused));
+            var retrieved = await _github.Repository.Comment.Get(_context.RepositoryOwner, _context.RepositoryName, result.Id);
+
+            Assert.True(retrieved.Id > 0);
+            Assert.Equal(1, retrieved.Reactions.TotalCount);
+            Assert.Equal(0, retrieved.Reactions.Plus1);
```

Comment:
i meant that we would setup an issue comment somewhere on github that has 5 users react with ":confused:" and 4 users react with ":heart:" and 3 users react with ":+1:" etc, then the integration test would check each of these.

```
var retrieved = await _github.Repository.Comment.Get("octokit", "octokit.net", commentId);

Assert.Equals(5, received.Reactions.Confused);
Assert.Equals(4, received.Reactions.Heart);
Assert.Equals(3, received.Reactions.Plus1);
etc
```

But it was just a thought and it's not a particularly good one because it cant be setup in code easily (as you would need up to 5 individual test users to be able to create the reactions) and even setting one up manually involves co-ordination of multiple people and is then also able to start failing if any other user ever added a reaction to it!

So let's just settle for the integration test
- creating a repository/issue/comment (you are already doing this)
- adding 1 reaction of each type to it (currently you only add some reactions and not others, so change this to a foreach Enum.GetValues loop)
- assert each reaction in the response payload had a count of 1 and total had a count of 6

at least that tests all fields deserialized ok